bitbucket.go

```diff
@@ -358,12 +359,12 @@ func (e RepositoryEnvironmentTypeOption) String() string {
 }
 
 type RepositoryEnvironmentOptions struct {
-	Owner           string `json:"owner"`
-	RepoSlug        string `json:"repo_slug"`
-	Uuid            string `json:"uuid"`
-	Name            string `json:"name"`
+	Owner           string                          `json:"owner"`
+	RepoSlug        string                          `json:"repo_slug"`
+	Uuid            string                          `json:"uuid"`
+	Name            string                          `json:"name"`
```

Comment:
Please use gofmt to fix indents.

Reply:
This is the result from gofmt. Gofmt add spaces to align the Code.

Reply:
I’m glad to hear that. Thank you for your confirmation.